docs/source/markdown/podman.1.md

```diff
@@ -336,9 +336,11 @@ Images are pulled under `XDG_DATA_HOME` when specified, otherwise in the home di
 
 Currently the slirp4netns package is required to be installed to create a network device, otherwise rootless containers need to run in the network namespace of the host.
 
+In certain enviroments like HPC (High Performance Computing) users are not allowed to take advantage of the additional UIDs and GIDs from the /etc/subuid and /etc/subgid systems.  Even in this environment you are allowed to use rootless podman with a single UID.  You need to set the `ignore_chown_errors` option in the /etc/containers/storage.conf or in ~/.config/containers/storage.conf. This option tells podman when pulling an image to ignore chown errors when it attempts to change a file in a container image to match the non root UID in the image. This means all files get saved as the user's UID. Note this could cause issues when running the container.
```

Comment:
```suggestion
In certain environments like HPC (High Performance Computing), users cannot take advantage of the additional UIDs and GIDs from the /etc/subuid and /etc/subgid systems.  However, in this environment, rootless Podman can operate with a single UID.  To make this work, set the `ignore_chown_errors` option in the /etc/containers/storage.conf or in ~/.config/containers/storage.conf files. This option tells Podman when pulling an image to ignore chown errors when attempting to change a file in a container image to match the non-root UID in the image. This means all files get saved as the user's UID. Note this could cause issues when running the container.
```